src/components/Header.jsx

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState } from "react"
+import Navbar from "./NavBar"
+import { getUsers } from '../api'
+
+const Header = (props) => {
+    const [userList, setUserList] = useState([])
+
+    useEffect(() => {
+      getUsers().then((res)=> {
+          setUserList(res.users)
+        })
+    },[])
+   
+const handleChange = (e) => {
+    userList.map((user)=>{
+        if(Object.values(user).includes(e.target.value)){
+          props.setGetUser(user)
+        }
+    })
+}
```

Comment:
Just as a heads up, this is not something we would expect you to include and we also aren't reviewing these bits of code that aren't part of the ticket.
BUT, this is YOUR Frontend Project and thus, you can definitely include these. Just make sure these extra curriculars don't get in the way of the functionality you're supposed to add!